README.md

```diff
@@ -117,6 +117,15 @@ Success! Data written to: development/webapp/cockpit_api_pass
 
 Note: if secret value is equal to existing one, write is skipped. `--force` flag can be used to overwrite.
 
+### Write file content
+```
+> treasury write development/webapp/credentials_yaml <filename> --file
+Success! Data written to: development/webapp/credentials_yaml
+```
+
+Note: if secret value is equal to existing one, write is skipped. `--force` flag can be used to overwrite. Stored file content in key is gzipped and converted to base64
```

Comment:
Does KMS sets any limits on key size? It would be worth mentioning the limit (if any) here.

Reply:
changed here: https://github.com/AirHelp/treasury/pull/88/commits/0c7355daba2016ddbe7c57ef9eb97c4e96cdc4d4